src/database/models/__init__.py

```
@@ -1,19 +1,37 @@
-from .mapper import mapper_registry
-from .log_mapper import Log  # noqa F401
-from .task_mapper import Task  # noqa F401
-from .user_mapper import User  # noqa F401
-from .event_mapper import Event  # noqa F401
-from .project_mapper import Project  # noqa F401
-from .association_tables import project_developers_table  # noqa F401
-from .availability_mapper import OfficeAvailability  # noqa F401
+from .mapper import mapper_registry, SHARED_TABLES, REPORTS_TABLES
+
+# Shared table definitions (managed by ocrdone-backend)
+from .user_model import User, user_table  # noqa F401
```

Ruff (0.14.14)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)